src/components/Footer.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import Contacts from './Contacts';
+import FundingPartners from './FundingPartners';
+
+const FooterBox = styled.footer`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 170px;
+  background-color: whitesmoke;
```

Comment:
It would be worth taking this color from themes rather than hard-coding it here.